vexim/config/authsite.php

```diff
@@ -2,42 +2,50 @@
 	include_once dirname(__FILE__) . '/variables.php';
 	include_once dirname(__FILE__) . '/functions.php';
 	include_once dirname(__FILE__) . '/httpheaders.php';
-	
+
 	# some session debugging code
 	#print_r($_SESSION);
-	
+
 	# confirm we have the necessary session variables
 	if (!isset($_SESSION['user_id'])
 		|| !isset($_SESSION['domain_id'])
 		|| !isset($_SESSION['crypt'])
-		)		
+		)
 	{
-		header ("Location: index.php?login=failed"); 
-		die(); 
+		header ("Location: index.php?login=failed");
+		die();
+	}
+
+	if ($_SESSION['username'] != "siteadmin"){
```

Comment:
I don't see what changed here. But once your are changing it, you can use `!==` instead of `!=` as well.